NeuroID/NeuroIDClass/Extensions/NIDUser.swift

```diff
@@ -159,6 +185,16 @@ public extension NeuroID {
      @param {String} [attemptedRegisteredUserId] - an optional identifier for the login
      */
     static func attemptedLogin(_ attemptedRegisteredUserId: String? = nil) -> Bool {
+//        Save log event
+        let scrubbedId = (attemptedRegisteredUserId != nil) ? scrubIdentifier(identifier: attemptedRegisteredUserId!) : "null"
```

Comment:
This line seems to be repeated a lot (determining if it's null or not), can/should we update the scrub identifier to accept a String? and then return "null" inside that function? Then we could call it inline for the log message?

Reply:
Added "null" string as default value if identifier is nil in scrubIdentifier call.